sycl/test-e2e/SYCLBIN/link_input.cpp

```diff
@@ -0,0 +1,24 @@
+//==-------------- link_input.cpp --- SYCLBIN extension tests --------------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+
+// REQUIRES: (opencl || level_zero)
```

Comment:
Is there a reason why the linking-related tests are `REQUIRES: opencl || level_zero` whereas the other tests are `XFAIL: cuda || hip`?